reqs/tests/import_reqs_tests.py

```diff
@@ -342,3 +343,20 @@ def test_fix_excel_decimals(blank_csv_file, ids, expected):
     assert len(reqs) == 2
     assert reqs[0].req_id == expected[0]
     assert reqs[1].req_id == expected[1]
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("requirement_id", ("", "100", "0" "1,000"), ids=repr)
+@pytest.mark.xfail(raises=ValueError)
```

Comment:
Nope, I was just experimenting with the decorator. I'll switch it to the context manager for consistency.